autoload/neoterm/repl.vim

```diff
@@ -24,12 +24,11 @@ function! neoterm#repl#term(id)
   if has_key(g:neoterm.instances, a:id)
     let g:neoterm.repl.instance_id = a:id
     let g:neoterm.repl.loaded = 1
-
```

Comment:
let's keep the diff with the minimum. Avoid unnecessary changes.

Reply:
Oh, sry.